include/CommonCrypto/CommonCryptor.h

```diff
@@ -28,6 +28,10 @@
 #define CC_SHA384_DIGEST_LENGTH 48
 #define CC_SHA512_DIGEST_LENGTH 64
 
+#ifndef __cplusplus
+typedef int8_t bool;
```

Comment:
Does it make more sense for us to include stdbool.h here? It does the same (similar: `bool -> _Bool` from C99) typedef in a C++ guard.